.vib/moodle/goss/goss.yaml

```diff
@@ -0,0 +1,12 @@
+file:
+  /bitnami/moodledata:
+    exists: true
+    filetype: directory
+    mode: "2775"
+  /bitnami/moodle/config.php:
```

Comment:
This test is not really necessary; if the deployment was successful a connection to the database was achieved.